### DISCUSSION TITLE
Incorrect request in the login page

### DISCUSSION THREAD

**Discussion by Ilamparithi-Tamizhanangu** [YetiForceCompany/YetiForceCRM#16736] [General] [answered]
Hi,

When i loging to the Yetiforce CRM, i am getting the below message. 

![image](https://user-images.githubusercontent.com/96602176/207511994-fdcd7f24-b26a-4f82-91b2-1306bcba3e04.png)

I am using Yetiforce 6.4 with PHP  8.0. 

It happens only the first time login for the particular day. When i click the dashboard button, it loads the dashboard page and everything working fine. 

It happens only first time login for the particular day. 

May i know where to check the error details. 

Thanks & regards,
Rajasekaran.

**Comment by rskrzypczak** [comment, marked as answer]
https://yetiforce.com/pl/component/zoo/item/debugging

**Reply by Ilamparithi-Tamizhanangu** [reply]
Thanks for the details.
